docs/dev/tvmdbg.md

```diff
@@ -0,0 +1,57 @@
+**TVMDBG**
+
+TVM Debugger (TVMDBG) is a UI based specialized debugger for TVM's computation graphs. It provides access to internal graph structures and tensor values at TVM runtime.
+
+**Why**  **TVMDBG**
+In TVM&#39;s current computation-graph framework, almost all actual computation after graph construction happens in a single Python function. Basic Python debugging tools such as pdb cannot be used to debug tvm.run, due to the fact that TVM&#39;s graph execution happens in the underlying C++ layer. C++ debugging tools such as [gdb](https://www.gnu.org/software/gdb/) are not ideal either, because of their inability to recognize and organize the stack frames and variables in a way relevant to TVM&#39;s operations, tensors and other graph constructs.
+
+TVMDBG addresses these limitations. Among the features provided by TVMDBG, the following ones are designed to facilitate runtime debugging of TVM:
+- Inspection of runtime ops output values and node connections
+- Time consumed by each fused operation
+
+**Note:** The TVM debugger uses a curses-based text user interface.
+
+**How to use TVMDBG?**
+1. In `config.cmake` set the `USE_GRAPH_RUNTIME_DEBUG` flag to `ON`
+```
+# Whether enable additional graph debug functions
+set(USE_GRAPH_RUNTIME_DEBUG ON)
+```
+2. Make tvm so that it will make the `libtvm_runtime.so`
+
+3. In the graph build file instead of `from tvm.contrib import graph_runtime` import the debug_runtime `from tvm.contrib.debugging import debug_runtime as graph_runtime`
+```
+#from tvm.contrib import graph_runtime
+from tvm.contrib.debugging import debug_runtime as graph_runtime
+```
+
+The above two modifications will bring up the debug UI during run.
+
+The HOME page of tvmdbg looks like below.
+ ![](https://raw.githubusercontent.com/siju-samuel/tvmdbg/master/docs/dev/_images/tvm_dbg1.png)
```

Comment:
@tqchen do we have some 'official' place to store these images?

Reply:
dmlc/web-data can be used for all content.

Reply:
Let us use https://github.com/tvmai/tvmai.github.io for this